app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -194,3 +194,8 @@ span.btn a{ text-decoration: none; color: #333333}
 body.modal-open {
   overflow: visible;
 }
+
+
+.glyphicon-copy:before {
+  content: "\f121";
```

Comment:
This was an attempt to use copy glyphicon, which is not supported in version 3.0.3 (our current version) - yet it didn't work. Removing this as eventually I've used paperclip glyphicon.